### ISSUE TITLE
Monthly Trends - "Any interactions" or "Any place"

### ISSUE THREAD

**Issue by ultimike** [communityinclusion/escoach#69]
From #58:

> Question: Are "Who=Any interaction" or "Where=Any place" valid selections for Monthly Trends? If so, we'll need new test data tables in Google Sheets to validate with.
> Answer: "Who = Any interaction" is the same as "Who = no selection". The same applies to the "Where = Anyplace" is the same as "Where = No selection". Rationale: With the "up-to-date" tables we needed the "Any interactions" and "Anyplace" queries to generate the summary tables for Who and Where. But for the trend data, the summary tables would be complicated and probably not very useful.

@escoach Understood - so should we add some code to popup a message if a user tries to select "Any interaction" or "Any place" when "Monthly Trends" or "Quarterly Trends" is selected?

-mike

**Comment by escoach** [comment]
@ultimike sure. good idea.
Could you add a pop-up that says "This selection does not apply to trend data" or, shorter: "Not applicable to trend data" or "Not available for trend data". 
It would need to be added to both the "No selection" for Who and Where and "Any interaction" and "Anyplace".

**Comment by ultimike** [comment]
@mgcobblestone ☝️ Make sense? If so, please implement 😄 

thanks,  
-mike

**Comment by mgcobblestone** [comment]
@ultimike Sure.  Just to be clear, other selections (besides none or any) are still valid for who/where?

**Comment by ultimike** [comment]
@mgcobblestone I believe so.

@escoach Please confirm.

-mike

**Comment by escoach** [comment]
@ultimike @mgcobblestone Correct. Other selections (besides none or any) are still valid for who/where

**Comment by mgcobblestone** [comment]
@ultimike @escoach Sorry, but just want to make sure I understand.  If No or Any are not valid, that means that there must be some other selection for BOTH who and where.  Is this correct?

**Comment by escoach** [comment]
@mgcobblestone @ultimike Thanks for asking. Your question made me take a closer look at the dev site as if I was a user and I realized that probably we do not need pop-up alerts.

For example, let's say I am a user and I make the following selections:
1) I select a "What" activity and nothing else. I get a table of time values.
2) Next, I keep the same "What" selection and I select "Who=A person I support". The data in the table change.
3) Next, I keep the "What" selection but I select "Who=No selection". The data return to scenario (1) above. No need for a pop-up. 
3bis) If instead of "Who=no selection" I selected "Who=Any interaction", again the table would show the same data as in scenario (1). I am not sure that an alert would be needed as it is obvious that "any interaction" means it does not matter whom the interaction was with.

The same applies for the "Where" drop-down menu.

It does look strange that there is both a "No selection" and "Any interaction" and both yield the same data. They make sense in the "up-to-date" format, not so much in the trend data. But I am not sure that the pop-up alerts help either. Let's hold off adding pop-up alerts until later if better easy solutions come up. Unless you have better technical suggestions that I am not aware of.
